generated/nifake/session.py

```diff
@@ -130,7 +130,7 @@ class _SessionBase(object):
     An property of type integer with read/write access.
     '''
     read_write_integer_with_converter = _attributes.AttributeViInt32TimeDeltaMilliseconds(1000008)
-    '''Type: datetime.timedelta
+    '''Type: int
```

Comment:
Can you correctly override the type_in_documentation on these so that it matches what a public API would say?

Reply:
They were overridden, but the template still used `'python_type'`. Fixed